Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -50,10 +50,16 @@
 
 # register matplotlib projection
 try:
-    from mantid import plots  # noqa
-    from mantid.plots._compatability import plotSpectrum, plotBin  # noqa
+    from mantid import plots
+    from mantid.plots._compatability import plotSpectrum, plotBin
 except ImportError:
-    pass  # matplotlib is unavailable
+
+    def notSupported(*args, **kwargs):
+        raise "Plotting is not supported by default. Please install matplotlib using: pip install matplotlib."
```

Comment:
```suggestion
        raise ImportError("Plotting is not supported by default. Please install matplotlib.")
```
Can't raise a string in python (`TypeError: exceptions must derive from BaseException`) and I don't think we should recommend using pip with our conda package, leave it up to them how to install mpl.

Reply:
Okay, Thanks for suggestion, I will add it